src/tic_tac_toe/read_write/timestamper.clj

```diff
@@ -0,0 +1,26 @@
+(ns tic-tac-toe.read-write.timestamper
+  (:use [clj-time.core :only [now interval in-days]]
+        [clj-time.format :as f]))
+
+(def date-formatter
+  (f/formatter "yyyyMMdd"))
+
+(defprotocol Timestamper
+  (current-date [this])
+  (date-difference [this updated]))
+
+(defrecord FSTimestamper []
```

Comment:
What is FS? I see there is also an `FSReader`

Reply:
There is an [`FSReader`](https://github.com/saralein/tic-tac-toe_clojure/blob/save-naming/src/tic_tac_toe/read_write/reader.clj#L14).  I named it `FSReader`, etc., for file system reader.  I suppose I was trying to avoid a very long name, but could also change it to be more descriptive.